src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                    .flatMap(str -> Arrays.stream(str.split(",")))
+                    .mapToInt(Integer::parseInt)
+                    .filter(number -> number % 2 == 0).min()
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/209573086-375eec70-c913-41db-8643-381526bfa120.png)

```suggestion
                    .filter(number -> number % 2 == 0)
                    .min()
```